src/classes/ADDR_Addresses_TDTM.cls

```diff
@@ -234,6 +243,13 @@ public with sharing class ADDR_Addresses_TDTM extends TDTM_Runnable {
 	                }      
 	            }
             }
+            
+            // finally, if no default found, create an empty default
```

Comment:
to support the scenario where the user is deleting the last address object.
 this will cause the account and contact address fields to get cleared,
which was reported as a bug (that they didn't get cleared).


David Habib | Senior Force.com Developer | salesforcefoundation.org

On Thu, May 21, 2015 at 3:46 PM, Nicolas Campbell <notifications@github.com>
wrote:

> In src/classes/ADDR_Addresses_TDTM.cls
> <https://github.com/SalesforceFoundation/Cumulus/pull/1573#discussion_r30855720>
> :
>
> > @@ -234,6 +243,13 @@ public with sharing class ADDR_Addresses_TDTM extends TDTM_Runnable {
> >  	                }
> >  	            }
> >              }
> > +
> > +            // finally, if no default found, create an empty default
>
> Why are we creating an empty default?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/SalesforceFoundation/Cumulus/pull/1573/files#r30855720>
> .
>